### DISCUSSION TITLE
Change project name

### DISCUSSION THREAD

**Discussion by doctor64** [doctor64/tuyaZigbee#30] [Ideas]
Recently, I get a suggestion to rename project to something less ambiguous. 
Proposed new name `Open Tuya Zigbee Firmware`
I personally really bad in naming, so I opened for ideas

PS: Taking into account my much less than ideal speed of work, I fear something like `Open but very slow implemented Tuya Zigbee Firmware` look like more appropriate :disappointed:

**Comment by MnM001** [comment]
OTZ (Open Tuya Zigbee)- sounds good to me :)

I personally would drop the firmware from the name.

**Reply by doctor64** [reply]
It can be confusing, because I don't change anything in hardware. Also it can be misleading because I can support only Telink based devices. I can, of course, support other chips, but it will require too much efforts from my side.
